### DISCUSSION TITLE
Method not allowed (Code 1203) - SOLVED

### DISCUSSION THREAD

**Discussion by drewbdoo** [homeassistant-apps/app-cloudflared#243] [Q&A] [answered]
Hi, I've been trying to set this up, but when I go to the link to authorize the tunnel and select my site and click authorize, I'm getting an error that says Method Not Allowed (Code 1203). As far as I can google, 1203 is just "something didn't work" and doesn't seem very helpful. Thanks for any help.

**Comment by brenner-tobias** [comment, marked as answer]
Where exactly do you get this error? Would be great if you could add a screenshot or more detailed explanation.

**Reply by drewbdoo** [reply]
I was a popup on cloudflare when I attempted to click authorize. I believe I have found the issue however and it was that there was a depreciated tunnel to the same application. Removing that through the dashboard has allowed me to complete the authorization, so I'm sure that was what it was. Thanks!
